ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs

```diff
@@ -238,6 +275,7 @@ private void UpdateTextBlockCore(TextBlock textBlock, bool measureOnly)
             textBlock.FontSize = _fontSize ?? 15;
             textBlock.FontStyle = _fontStyle ?? new FontStyle();
             textBlock.FontWeight = _fontWeight ?? FontWeights.Normal;
+            textBlock.TextDecorations = new TextDecorationCollection(TextDecorationCollection);
 
```

Comment:
This would be supported on [Android](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextShadowNode.java#L161-L166) and iOS:
```
<Text>
   Here is some regular text <Text style={{textDecorationLine: 'underline'}}>and here is some underlined virtual text</Text>.
</Text>
```



However, this scenario would not supported by this PR.

---
In reply to: [105127837](https://github.com/Microsoft/react-native-windows/pull/1056#discussion_r105127837) [](ancestors = 105127837)